ci-operator/step-registry/ignition/bastionhost/ignition-bastionhost-commands.sh

```diff
@@ -219,7 +219,16 @@ read only = false
 timeout = 300
 EOF
 
-RSYNCD_CONFIG_CONTENT=$(cat ${workdir}/rsyncd.conf | base64 -w0)
+#ibmcloud fail to create the bastion with the encoded RSYNCD_CONFIG_CONTENT, blocked by cloudflare
+if [[ "${CLUSTER_TYPE}" == "ibmcloud" ]]; then
```

Comment:
Can you prepare a ibmcloud bastion using this ignition file to let us double confirm ?

Reply:
https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/65506/rehearse-65506-periodic-ci-openshift-verification-tests-main-installer-rehearse-4.20-installer-rehearse-ibmcloud-dis/1930490283154214912 @jianlinliu

Reply:
How can I get the bastion IP ?

Reply:
Are you sure `source` format without `data:text/plain;base64` can work in ignition world ?

Reply:
The bastion vm has error when "open vnc console" from https://cloud.ibm.com/infrastructure/compute/vs/us-east-3~0777_7d26005b-b6c8-4eeb-a427-8a8833be9012/overview, I'm checking